.github/ISSUE_TEMPLATE/docs.yml

```
@@ -0,0 +1,43 @@
+name: "\U0001F4DA Documentation"
+description: Suggest a change or new page to be added to vitejs.dev
+labels: [documentation]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this issue!
+  - type: checkboxes
+    id: documentation_is
+    attributes:
+      label: Documentation is
+      options:
+        - label: Missing
+        - label: Needed
+        - label: Confusing
+        - label: Not sure?
```

I think "missing" and "needed" is quite synonymous in their usecase. Maybe we could replace "Needed" as "Outdated" instead?
```suggestion
        - label: Missing
        - label: Outdated
        - label: Confusing
        - label: Not sure?
```

---

Ya, that looks nicer. 👍